### ISSUE TITLE
Translate Play Store listing

### ISSUE THREAD

**Issue by nicolas-raoul** [commons-app/apps-android-commons#790]
To lower the barrier to entry for non-English readers, we should translate the store listing to other languages:
![screenshot from 2017-07-18 17-26-26](https://user-images.githubusercontent.com/99590/28307643-83f037b2-6bde-11e7-9f56-1752ffc21d8f.png)

And maybe even these (lower priority, but if someone makes screenshots/videos in other languages please send them to us, we can upload them, and create a separate Github repository to keep track of them and make them easily usable on other app stores):
![screenshot from 2017-07-18 17-26-52](https://user-images.githubusercontent.com/99590/28307760-ec2fc31a-6bde-11e7-8499-c13c71a4e16d.png)

**Comment by misaochan** [comment]
I agree, would be very useful. Is there a way to integrate this into translatewiki?

**Comment by nicolas-raoul** [comment]
Yes, just create a new file containing these string, in any format that the
tool accepts.

**Comment by nsaiisasidhar** [comment]
Hi, can I work on this enhancement.

**Comment by nicolas-raoul** [comment]
@nsaiisasidhar actually you don't need to translate anything, what needs to be done is just to create the English-language file, and modify the TranslateWiki configuration.

**Comment by nsaiisasidhar** [comment]
Sure. I'll do it soon. Thank you

**Comment by nicolas-raoul** [comment]
The files are now at https://github.com/commons-app/apps-android-commons/tree/master/marketing
What is left is to figure out how to have Translatewiki translates them as well.

(In a later phase we could break down the long description so that the whole translations are not lost whenever we change anything in the English original)

**Comment by nicolas-raoul** [comment]
The translatewiki configuration file is actually this file:
https://phabricator.wikimedia.org/diffusion/GTWN/browse/master/groups/Wikimedia/CommonsAndroid.yaml

**Comment by Clockery** [comment]
Is this the file?
https://github.com/wikimedia/translatewiki/blob/master/groups/Wikimedia/CommonsAndroid.yaml

**Comment by Clockery** [comment]
Oh, wait, I didn't see your earlier comment :) Aren't both of them the same file?

**Comment by nicolas-raoul** [comment]
Yes!
Github is only a mirror of Phabricator, so pull requests are better sent to Phabricator.

**Comment by nicolas-raoul** [comment]
Yes, you can get an understanding of the syntax by checking the many examples at https://phabricator.wikimedia.org/diffusion/GTWN/browse/master/groups/

**Comment by Clockery** [comment]
I'm not really sure how to send a pull request on Phabricator... I'm using Differential to create a diff right now. Does the "Repository" field refer to the file itself, or the folder which contains the file?

https://phabricator.wikimedia.org/differential/diff/create/

**Comment by nicolas-raoul** [comment]
That Git repo is outside of our control and they usually are busy, so just
post your modified file when you are sure about it 🙂

**Comment by Clockery** [comment]
I've made the changes (which I hope are okay), where do I post it to?

**Comment by nicolas-raoul** [comment]
Here 🙂

**Comment by Clockery** [comment]
[diff.txt](https://github.com/commons-app/apps-android-commons/files/1559690/diff.txt)

Here's the file, please let me know if anything should be changed. I hope it's all right :)

**Comment by nicolas-raoul** [comment]
That would overwrite the English version.
Please suffix with %CODE%

**Comment by Clockery** [comment]
Sure, but where should the suffix be added? Do I add it as a separate field under MANGLER?

**Comment by Clockery** [comment]
No, wait, I think I get it. As part of the links, right?

**Comment by Clockery** [comment]
[diff.txt](https://github.com/commons-app/apps-android-commons/files/1559738/diff.txt)

Fixed it now (I hope).

**Comment by Clockery** [comment]
Here's the fixed (renamed) file

[diff-revised.txt](https://github.com/commons-app/apps-android-commons/files/1559813/diff-revised.txt)

**Comment by nicolas-raoul** [comment]
Last thing, the id and label need to be different for each file.

**Comment by Clockery** [comment]
[diff-revised-2.txt](https://github.com/commons-app/apps-android-commons/files/1559926/diff-revised-2.txt)

Is this all right?

**Comment by nicolas-raoul** [comment]
Yes looks good, thanks! :-)

**Comment by Clockery** [comment]
Thank you so much! :D

**Comment by nicolas-raoul** [comment]
https://translatewiki.net/wiki/Thread:Support/Adding_3_text_files_to_Commons_Android

**Comment by nicolas-raoul** [comment]
https://gerrit.wikimedia.org/r/#/c/399355/

**Comment by nicolas-raoul** [comment]
Looks like something messed up, and my patch did not make it into the YAML file:
https://phabricator.wikimedia.org/diffusion/GTWN/browse/master/groups/Wikimedia/CommonsAndroid.yaml
Could someone try again?

**Comment by m-alzam** [comment]
Hi,
Could you add this as a Google Code-in task?

**Comment by nicolas-raoul** [comment]
Unfortunately CGI rules currently exclude pure-localization tasks :-/

**Comment by m-alzam** [comment]
Oh, thanks. Could you remove the `code-in` label, so others students don't go here thinking about the same?
